plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveSplitManager.java

```diff
@@ -221,9 +227,6 @@ public ConnectorSplitSource getSplits(
         // validate bucket bucketed execution
         Optional<HiveBucketHandle> bucketHandle = hiveTable.getBucketHandle();
 
-        // sort partitions
```

Comment:
Don't we have to sort them ?

Reply:
It looks like it is not necessary. It could be that case that the partitions were sorted for debugging convenience reasons. @electrum  / @dain  do you remember why could it be necessary?